### ISSUE TITLE
RPM build errors when installing tests folder

### ISSUE THREAD

**Issue by cyrilleverrier** [linux-rdma/ibsim#8]
How to reproduce:

```
docker run --name centos7 -v `pwd`:/ibsim -i -t --rm centos:7 bash 

yum group install --assumeyes  "Development Tools"
yum install --assumeyes        rpmdevtools
yum install --assumeyes        libibmad-devel libibumad-devel
mkdir -p /source
rpmdev-setuptree

cd /ibsim
IBSIM_VERSION=`cat ibsim/ibsim.c | sed -ne '/#define IBSIM_VERSION /s/^#define IBSIM_VERSION \"\(.*\)\"/\1/p'`
make dist RELEASE=1
tar -xOzf ibsim-${IBSIM_VERSION}.tar.gz ibsim-${IBSIM_VERSION}/ibsim.spec  > ibsim.spec
yes | cp -f ibsim-${IBSIM_VERSION}.tar.gz /root/rpmbuild/SOURCES/
rpmbuild -ba ibsim.spec
```

Output:
```
make -C ibsim install &&   make -C umad2sim install &&   make -C tests install &&  echo "Done."
make[1]: Entering directory `/root/rpmbuild/BUILD/ibsim-0.8/ibsim'
install -d /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin
install -d /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/lib64/umad2sim
install ibsim /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin
make[1]: Leaving directory `/root/rpmbuild/BUILD/ibsim-0.8/ibsim'
make[1]: Entering directory `/root/rpmbuild/BUILD/ibsim-0.8/umad2sim'
install -d /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin
install -d /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/lib64/umad2sim
install libumad2sim.so /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/lib64/umad2sim
make[1]: Leaving directory `/root/rpmbuild/BUILD/ibsim-0.8/umad2sim'
make[1]: Entering directory `/root/rpmbuild/BUILD/ibsim-0.8/tests'
install -d /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin
install -d /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/lib64/umad2sim
install subnet_discover /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin  install query_many /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin  install mcast_storm /root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin
install: omitting directory '/root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin'
install: cannot stat 'install': No such file or directory
install: omitting directory '/root/rpmbuild/BUILDROOT/ibsim-0.8-1.el7.x86_64/usr/bin'
install: cannot stat 'install': No such file or directory
make[1]: *** [install] Error 1
make[1]: Leaving directory `/root/rpmbuild/BUILD/ibsim-0.8/tests'
make: *** [install] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.2AgKKI (%install)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.2AgKKI (%install)
```

Since https://github.com/linux-rdma/ibsim/commit/1852e317ee4de48dce513de3f7c401d5a42bd95f

**Comment by tamirronen** [comment]
Thanks, patch merged to master (commit 9bd8b8a)

**Comment by tamirronen** [comment]
Thanks, patch merged to master (commit 9bd8b8a)
